TwilioVerify/TwilioVerify/Sources/Domain/Factor/PushFactory.swift

```diff
@@ -43,6 +43,7 @@ extension PushFactory: PushFactoryProtocol {
   func createFactor(withAccessToken accessToken: String, friendlyName: String, pushToken: String, serviceSid: String,
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/379e60024e59da8f797b45da3b7c4c930c28e6de/TwilioVerify/TwilioVerify/Sources/Domain/Factor/PushFactory.swift#L43"></span>Function body should span 40 lines or less excluding comments and whitespace: currently spans 43 lines<br />
<code>function_body_length</code> <code>PushFactory.swift:43</code></td>
    </tr>
  </tbody>
</table>